checkov/common/checks_infra/solvers/resource_solvers/base_resource_solver.py

```diff
@@ -49,6 +49,8 @@ def run(
 
         for _, data in graph_connector.nodes():
             result = self.get_operation(resource_type=data.get(CustomAttributes.RESOURCE_TYPE))
+            if result is None:
+                continue
             if result:
```

Comment:
why change the logic? `if result is None:` and the preview `if result:` is the same.
you can just not add to the passed_vertices if we have the result (it means it is not the resource in the policy).

Reply:
because I wanted to have an option of `None` result which indicates that the resource "passed", and in this flow it means it should be skipped.
pls pay attention that `base_resource_solver` `run` and `_process_node` functions handle both `exists_resource_solver` and `not_exists_resource_solver`.
In `ExistsResourcerSolver.get_operation ` we can get either True/False, while in `NotExistsResourcerSolver.get_operation` we can get False/None